common/templates/layouts/base.njk

```diff
@@ -65,7 +65,7 @@
     containerClasses: "govuk-width-container",
     navigation: [
       {
-        text: USER.userName
+        text: USER.firstName + " " + USER.lastName
```

Comment:
Might be neater to add a `fullname` property onto the User class when it is built so that the fullname can be referenced anywhere else and we know it will always be consistent.

For example, it could be re-used in the future on a welcome page or a user profile page.